pages/solutions/census-alternative.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import { Container, Button, Row, Col } from "react-bootstrap";
+import Head from "next/head";
+import GetStarted from "../../components/home/getStarted";
+import hydrate from "next-mdx-remote/hydrate";
+import { loadMdxFile } from "../../utils/mdxUtils";
+
+const components = {};
+
+export default function VsCensus({ pageProps }) {
```

Comment:
I'm wondering how similar this is with the Segment page? I mentioned sharing a section below, but now I'm wondering if this should be a templated approach?

IOW building a template at `pages/solutions/[solution].tsx` and then using individual MDX files for each solution. And we can use frontmatter for some of more unique sections.

Maybe that's overkill at this point, but my gut is (without looking) that a lot of the code was duplicated.

Reply:
I like that idea better.  I'd built out separate components for the reusable parts, but reverted that commit because it should really be a separate PR.  A template sounds like a tidier solution overall.  I'll put that together instead and then start working on visual elements (checkboxes, table, etc.)